### PR TITLE
[feg/lte/xwf] upgrade dockers to focal with py3.8

### PR DESCRIPTION
**Pull request by AyliD**
[feg/lte] upgrade dockers to focal with py3.8

## Summary

Changed docker files for feg to use py 3.8 since py3.5 is deprecated

NOTE:
Do not merge this till we get supported version for focal of package magma-nghttp, otherwise docker control_proxy wont start

## Test Plan

```
$ cd magma/feg/gateway/docker
$ docker-compose build
$ docker-compose up -d

$ docker ps
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS              PORTS               NAMES
9696160acd0e        feg_gateway_python:latest   "python3.8 -m magma.…"   9 seconds ago       Up 8 seconds                            magmad
bfe342db5f0a        feg_gateway_python:latest   "/bin/bash -c '/usr/…"   9 seconds ago       Up 8 seconds                            control_proxy
ce17b47a13eb        feg_gateway_python:latest   "/bin/bash -c '/usr/…"   10 seconds ago      Up 9 seconds                            redis
35dd0ea7dc2d        feg_gateway_go_base         "/bin/bash -c 'mkdir…"   10 seconds ago      Up 9 seconds                            test
6fe0c8b1e2f7        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 17 seconds                           radiusd
2a3c08fcee67        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 17 seconds                           eap_aka
0a758bf6f40f        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 17 seconds                           csfb
d71bfd0f9ab8        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 17 seconds                           swx_proxy
04b7f849bb1c        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 17 seconds                           s6a_proxy
2d3a0a7dd66c        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 18 seconds                           feg_hello
2729e89fe120        feg_gateway_go              "envdir /var/opt/mag…"   19 seconds ago      Up 18 seconds                           hss
07db950315a6        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 18 seconds                           aaa_server
9fa5ad5e3fad        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 18 seconds                           session_proxy
d8bf7ec7536c        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 18 seconds                           health
44d07356ee0c        feg_gateway_go:latest       "envdir /var/opt/mag…"   19 seconds ago      Up 18 seconds                           eap_sim
b7c70da83730        feg_gateway_python:latest   "python3.8 -m magma.…"   20 seconds ago      Up 19 seconds                           eventd
62c6344846dd        feg_gateway_python:latest   "/bin/bash -c '/usr/…"   20 seconds ago      Up 18 seconds                           td-agent-bit

```

Tested also xwf-m box with cwf's pipelined and magmad and made sure all looks ok and running py3.8.

```
# docker ps
CONTAINER ID   IMAGE                                                                    COMMAND                  CREATED         STATUS                   PORTS                    NAMES
36bacfa1cc69   facebookconnectivity-xwf-m-staging-docker.jfrog.io/goradius:aff7160f     "/bin/sh -c ./docker…"   3 minutes ago   Up 3 minutes                                      radius
519c5017099d   facebookconnectivity-openflow-docker.jfrog.io/gateway_pipelined:latest   "sh -c 'set bridge c…"   3 minutes ago   Up 3 minutes                                      pipelined
7ccc808fe098   facebookconnectivity-openflow-docker.jfrog.io/gateway_python:latest      "python3.8 -m magma.…"   3 minutes ago   Up 3 minutes                                      magmad
8fa7d6eec5b1   facebookconnectivity-openflow-docker.jfrog.io/gateway_python:latest      "sh -c '/usr/local/b…"   3 minutes ago   Up 3 minutes                                      control_proxy
92bd4591c73a   facebookconnectivity-xwf-m-staging-docker.jfrog.io/gateway_go:aff7160f   "envdir /var/opt/mag…"   3 minutes ago   Up 3 minutes                                      radiusd
2f579a7c1bb1   facebookconnectivity-xwf-m-staging-docker.jfrog.io/logrouter             "/bin/sh -c /entrypo…"   3 minutes ago   Up 3 minutes (healthy)                            logrouter
2f63909df67a   gcr.io/google-containers/cadvisor:latest                                 "/usr/bin/cadvisor -…"   3 minutes ago   Up 3 minutes (healthy)   0.0.0.0:8080->8080/tcp   cadvisor

```
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: cwf/gateway/docker/docker-compose.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@ services:
       test: ["CMD", "nc", "-zv", "localhost","50067"]
       timeout: "4s"
       retries: 3
-    command: python3.5 -m magma.directoryd.main
+    command: python3.8 -m magma.directoryd.main
```
Comment (pshelar) [inline cwf/gateway/docker/docker-compose.yml:70]: On orc8r we are completely dropping support for 3.5. we also need to update documentation.

@karthiksubraveti @mpgermano what do you guys think.

File: cwf/gateway/docker/python/Dockerfile [hunk 2]
```diff
@@ -50,17 +53,17 @@ RUN apt-key add /tmp/jfrog.pub && \
     apt-add-repository "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev/ bionic main" && \
```
Comment (tmdzk) [inline cwf/gateway/docker/python/Dockerfile]: We should start pushing as distribution=focal and change s/bionic/focal here

File: cwf/gateway/docker/python/Dockerfile [hunk 3]
```diff
@@ -50,17 +53,17 @@ RUN apt-key add /tmp/jfrog.pub && \
...
     apt-add-repository "deb http://archive.ubuntu.com/ubuntu/ bionic-proposed restricted main multiverse universe"
```
Comment (tmdzk) [inline cwf/gateway/docker/python/Dockerfile]: Same here we should point to the `focal-proposed`

File: feg/gateway/docker/python/Dockerfile [hunk 4]
```diff
@@ -72,9 +75,9 @@ RUN curl -L http://packages.fluentbit.io/fluentbit.key > /tmp/fluentbit.key
 RUN apt-key add /tmp/fluentbit.key && \
     apt-add-repository "deb https://packages.fluentbit.io/ubuntu/bionic bionic main"
```
Comment (tmdzk) [inline feg/gateway/docker/python/Dockerfile]: same here. 
`deb https://packages.fluentbit.io/ubuntu/focal focal main`

**Review by tmdzk — COMMENTED** [review]
The overall looks good. Just a few comments.

File: cwf/gateway/docker/python/Dockerfile (hunk 2 above)
Comment (AyliD) [reply cwf/gateway/docker/python/Dockerfile]: Currently there is no such release so this cause failure in build process.

**Comment by AyliD** [comment]
https://github.com/magma/magma/pull/4450 instead.

**Pull request closed without merging**